src/components/views/UserPage.js

```diff
@@ -1,6 +1,67 @@
-const UserPage = () => {
-    return <div>UserPage</div>
+import React, {useEffect, useState} from "react";
+import Header from "./Header";
+import {api} from 'helpers/api';
+import BaseContainer from "../ui/BaseContainer";
+import 'styles/views/UserPage.scss'
+
+
+const PlayerProfile = ({user}) =>(
+    <div className="userPage container">
+        <ul className="userPage player-info-container">Username: {user.username}</ul>
+        <ul className="userPage player-info-container">Name: {user.name}</ul>
+        <ul className="userPage player-info-container">Gender: {user.gender}</ul>
+        <ul className="userPage player-info-container">Birthday: {displayDate(user.birthday)}</ul>
+    </div>
+)
+const displayDate = (date) => {
+    if(date){
```

Comment:
Is there no other possibility to parse the date than to take the substrings? Seems a bit risky in case of changes to me.

Reply:
Done, better now. (For some reason the date has to be made to a new date before it works, that is why 
const dt = new Date(date) line)